.github/workflows/npm-bru-cli.yml

```
@@ -1,51 +1,51 @@
-name: Bru CLI Tests (npm)
+# name: Bru CLI Tests (npm)
```

actionlint (1.7.12)

[error] workflow is empty

(syntax-check)